camOverlayRemoteControl/html/index.js

```diff
@@ -77,6 +80,7 @@ function addField(){
   $("#fieldContainer").html(genFields(field_count));
   populateFields(fields);
   $(".form-control").off();
+  $('#camProtocol').change(protocolChanged);
```

Comment:
Je tohle spravne misto? Podle me protocol neni per field ale jeden pro cele UI, takze by se mel callback setnout jen jednou na zacatku. Kdyby jsi to setoval pokazdy tak ti chybi off() viz radek nad a mnozily by se ti callbacky..